app/main.py

```diff
@@ -1,6 +1,25 @@
+import os
+
+import requests
+
+
+URL = "http://api.weatherapi.com/v1/current.json"
+API_KEY = os.environ["API_KEY"]
```

Comment:
Ensure that the `API_KEY` environment variable is set before running the script. Consider adding error handling to manage cases where the environment variable might not be set.